### PR TITLE
fix windows deployment and dependencies

### PR DESCRIPTION
**Pull request by anne-gropler**
fix: make qwindows.dll available via CMake variables
fix: newer GLM versions do not initialize matrix with identity
fix: handle Qt Spontaneous Events
fix: small adaptions in shader code and globjects::init, necessary for some drivers

### CODE REVIEW

File: source/llassetgen-rendering/main.cpp [hunk 1]
```diff
@@ -69,10 +69,10 @@ class Window : public WindowQt {
 
     virtual ~Window() {}
 
-    virtual void initializeGL() override {
+    virtual bool initializeGL() override {
         globjects::init([this](const char * name) {
             return getProcAddress(name);
-        });
+        }, globjects::Texture::BindlessImplementation::Legacy);
```
Comment (scheibel) [inline source/llassetgen-rendering/main.cpp]:
```suggestion
        });
```

File: source/llassetgen-rendering/main.cpp [hunk 2]
```diff
@@ -195,7 +197,7 @@ class Window : public WindowQt {
         program->setUniform("showDistanceField", showDistanceField);
         program->setUniform("superSampling", superSampling);
         program->setUniform("threshold", dtThreshold);
-
+        vao->bind();
```
Comment (scheibel) [inline source/llassetgen-rendering/main.cpp]:
```suggestion
```

File: source/llassetgen-rendering/main.cpp [hunk 3]
```diff
@@ -333,7 +335,7 @@ class Window : public WindowQt {
     }
 
     virtual void resetTransform3D() override {
-        transform3D = glm::mat4();  // set identity
+        transform3D = glm::mat4(1.0);  // set identity
```
Comment (scheibel) [inline source/llassetgen-rendering/main.cpp]:
```suggestion
        transform3D = glm::mat4(1.0f);  // set identity
```

File: source/llassetgen-rendering/main.cpp [hunk 4]
```diff
@@ -408,7 +410,7 @@ class Window : public WindowQt {
     glm::vec4 backgroundColor = glm::vec4(1.f, 1.f, 1.f, 1.f);
     glm::vec4 fontColor = glm::vec4(0.f, 0.f, 0.f, 1.f);
     int samplerIndex = 0;
-    glm::mat4 transform3D = glm::mat4();
+    glm::mat4 transform3D = glm::mat4(1.0);
```
Comment (scheibel) [inline source/llassetgen-rendering/main.cpp]:
```suggestion
    glm::mat4 transform3D = glm::mat4(1.0f);
```

File: source/llassetgen-rendering/main.cpp [hunk 5]
```diff
@@ -69,10 +69,10 @@ class Window : public WindowQt {
 
     virtual ~Window() {}
 
-    virtual void initializeGL() override {
+    virtual bool initializeGL() override {
         globjects::init([this](const char * name) {
```
Comment (scheibel) [inline source/llassetgen-rendering/main.cpp]: With current glbinding master, this line will get a problem.
Have a look at https://github.com/cginternals/glbinding/compare/8c18e53faffc4bfda11901c573a5c4bb0fc0b2d3...beac98e46babb819d225dee7bfd9b13774569351 for a workaround.

Comment (anne-gropler) [reply source/llassetgen-rendering/main.cpp]: What exactly is the problem? This is `globjects::init`, which does not appear in your workaround. Please help me to understand.

Comment (scheibel) [reply source/llassetgen-rendering/main.cpp]: This language construct dismisses the support for MSVC 2013. As we want to support this compiler further, we were forced to degrade to plain C function pointers for the lookup callback. Unfortunately, the present implementation requires a binding of `this` into the context of the callback that is not possible with plain C function pointers. A workaround is the usage of a global/static variable (as outlined in my reference to glbinding).

Comment (anne-gropler) [reply source/llassetgen-rendering/main.cpp]: thank you for this explanation, very helpful :)

File: source/llassetgen-rendering/main.cpp (hunk 1 above)
Comment (anne-gropler) [reply source/llassetgen-rendering/main.cpp]: @scheibel Just FYI: When I remove this line, I get this error:
```
#fatal: error: 0x502, high severity (API)
Error has been generated. GL error GL_INVALID_OPERATION in (null): (ID: 173538523) Generic error
```
on my machine with those specifications:
```
OpenGL Version:  4.3
OpenGL Vendor:   Intel
OpenGL Renderer: Intel(R) HD Graphics 520
```

However, everything renders fine, so I stay with the removed line.

**Pull request merged**